docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -1,30 +1,106 @@
 = Configuring dependencies rebuild for Java applications
 
-For Java applications in {ProductName}, you can configure the Java Virtual Machine (JVM) build service either with the dependencies rebuild feature enabled or with that feature disabled. This document outlines, for each scenario, why and how you would configure the JVM build service.  
+For Java applications in {ProductName}, you can configure the Java Virtual Machine (JVM) build service either with the dependencies rebuild feature enabled or with that feature disabled.
 
-== Configuring Java builds with enabled dependencies rebuild
+== JVM Build Service Overview
```

Comment:
== JVM Build Service overview

Note, you don't need to bold it. the `==` will make it a bigger, more noticeable font.